wooting-macro-backend/src/lib.rs

```diff
@@ -552,8 +543,6 @@ impl MacroBackend {
                         rdev::EventType::ButtonRelease(button) => {
                             debug!("Button released: {:?}", button);
 
-                            buttons_pressed.blocking_write().retain(|x| *x != button);
-
```

Comment:
?

Reply:
the variable was never used for anything useful as you can only have one mouse button pressed at a time and that is already provided by the event. It was originally meant to be there to check multiple combinations of buttons or key presses before we decided to drop it. Thus, in one of the branches i noticed this not being needed and removed it but during the merge it got back in